mspray/apps/main/views/performance.py

```diff
@@ -267,6 +268,7 @@ def get_context_data(self, **kwargs):
         context = super(SprayOperatorSummaryView, self).get_context_data(
             **kwargs
         )
+        community_health_worker = context["community_health_worker"]
```

Comment:
If we are using a "global context processor" is it still necessary to the change the individual context of a view in this way?  Isn't avoiding that why we chose a global context processor?